apps/wakunode2/external_config.nim

```diff
@@ -256,6 +258,11 @@ type
         desc: "Default pubsub topic to subscribe to. Argument may be repeated."
         name: "pubsub-topic" .}: seq[string]
 
+      shards* {.
```

Comment:
Thinking that in future the ability to configure ranges may be needed for large subscriptions (e.g. `--shards:0..7,15,21..29`). For now repeated arg makes sense.

Reply:
mm i would say node operators don't necessarily need to specify the exact shards but the number of them they want to run. unless they are interested in a specific shard because their content topic falls in there, but in that case i expect them to provide the contentTopic and not the shard.

(but yeap, current behaviour is --shard=5 means shard 5 and not 5 shards)